src/Microsoft.Dnx.Runtime/NuGet/Utility/PathUtility.cs

```diff
@@ -72,7 +73,72 @@ public static string GetRelativePath(string path1, string path2)
         /// </summary>
         public static string GetRelativePath(string path1, string path2, char separator)
         {
-            return UriUtility.GetPath(GetRelativeUri(path1, path2), separator);
+            if (path1 == null)
```

Comment:
The function needs absolute paths, what does it even mean to get a relative path from two relative paths? Its not possible without assuming they have the same base path.
If string.Empty is passed it should probably error too?